app/models/spree/content_item_decorator.rb

```diff
@@ -0,0 +1,9 @@
+module Spree
+  module Stock
+    ContentItem.class_eval do
+      def variant_has_product_packages?
```

Comment:
Good point about the prefix, the goal is so we don't have to do something like content_item.variant.product.product_packages, it works like the backordered? or on_hand? scope on ContentItem to basically just filter the results